pipeline/pipeline.py

```diff
@@ -71,4 +73,10 @@ def _run_task(self, task: AbstractTask):
             task.run()
         end = timer()
         logging.info(f'{task.name} Completed. Total time taken: {str(timedelta(seconds=end - start))}')
+        failed_task_log = logging_utils.get_error_log_file(task.action_type, task.object_type, self._working_dir)
+        # TODO: Add logic to skip failure check
```

Comment:
I think the pipeline should always fail if there are some failures. If the invoker for sure knows that he/she can skip the failure, then the user can simply skip that task via --skip-tasks.

btw, can we make sure the task is not checkpointed if the pipeline throws due to the error log files?

Reply:
yeah verified that